python/app/g2panel

```diff
@@ -418,15 +429,25 @@ class MainWindow(QtWidgets.QMainWindow):
     def triggerMode(self, selected):
```

Comment:
I think here we also need to update the receiver for the number of frames. Scenario is that the user switches between trigger and auto without editing the number of frames.

Reply:
I have added a change in frames for both auto and trigger mode